includes/Vendor/SetupWizard.php

```diff
@@ -666,4 +653,86 @@ public function dokan_setup_ready() {
         </div>
         <?php
     }
+
+    /**
+     * Gets the URL for the next step in the wizard
+     *
+     * Handles special logic to skip the payment step if no withdrawal methods
+     * are active, preventing users from accessing an empty payment step
+     *
+     * @since 2.9.27
+     *
+     * @return string The URL for the next step
+     */
+    public function get_next_step_link(): string {
+        $keys = array_keys( $this->steps );
+        $step = array_search( $this->step, $keys, true );
+        $next_step = $keys[ $step + 1 ];
```

Comment:
- Just increase the next step index `$step++` here and inside the `if` condition.
- Assign `$next_step` after if condition.